recipes/jupyterlab_miami_nights/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab_miami_nights" %}
+{% set version = "0.3.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_miami_nights-{{ version }}.tar.gz
+  sha256: 6789a03146945be80f138afd84d16b89d2f5ea5b2e5160c877b562974d436be7
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.10,<2
+  run:
+    - jupyterlab ~=3.0
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/@timkpaine/jupyterlab_miami_nights/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\@timkpaine\\jupyterlab_miami_nights\\package.json (exit 0) else (exit 1)          # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/timkpaine/jupyterlab_miami_nights
+  summary: Combination of VS Code's SynthWave '84 and JupyterLab's Neon Night 
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - timkpaine
```

Comment:
cc @timkpaine if you are fine to be listed here

Reply:
Yep